src/rest/rest-router.ts

```diff
@@ -46,6 +47,8 @@ subRouter.use("/installation", GitHubAppsRoute);
 
 subRouter.use("/jira/cloudid", JiraCloudIDRouter);
 
+subRouter.use("/subscription/:subscriptionId", SubscriptionDeleteRouter);
```

Comment:
```suggestion
subRouter.use("/subscriptions/:subscriptionId", SubscriptionRouter);
```

Reply:
I think we should use a one-level higher router here, coz: later if we want to do other operation on that subscription, we can't
```
subRouter.use("/subscription/:subscriptionId", SubscriptionDeleteRouter);
subRouter.use("/subscription/:subscriptionId", SubscriptionOtherOpsRouter);
```
Coz the route conflcts.

Reply:
Re-structured the routes.